installation/src/Model/CleanupModel.php

```diff
@@ -33,15 +34,48 @@ class CleanupModel extends BaseInstallationModel
      */
     public function deleteInstallationFolder()
     {
-        $return = Folder::delete(JPATH_INSTALLATION) && (!file_exists(JPATH_ROOT . '/joomla.xml') || File::delete(JPATH_ROOT . '/joomla.xml'));
+        // First, we try to delete the installation folder
+        try {
+            Folder::delete(JPATH_INSTALLATION);
+        } catch (FilesystemException $e) {
+            /**
+             * Windows quirk: The installation folder may fail to delete because
+             * index.php, though already deleted, remains locked by PHP until
+             * the request ends. If no subfolders and only that file is present,
+             * we can assume the deletion effectively successful and continue cleanup.
+             */
+            if (PHP_OS_FAMILY == 'Windows') {
```

Comment:
```suggestion
            if (PHP_OS_FAMILY === 'Windows') {
```
Reading https://www.php.net/manual/en/reserved.constants.php I think we can be sure it's a string so we can use strict comparison.